pushtemplates/src/main/java/com/clevertap/pushtemplates/VideoActivity.java

```diff
@@ -64,10 +64,29 @@ protected void onCreate(Bundle savedInstanceState) {
 
 
 		aspectRatioFrameLayout=findViewById(R.id.video_activity);
-		FrameLayout.LayoutParams params = (FrameLayout.LayoutParams)  videoView.getLayoutParams();
-		params.width = (int) (Constants.PT_VIDEO_WIDTH * getApplicationContext().getResources().getDisplayMetrics().density);
-		params.height = (int) ( Constants.PT_VIDEO_HEIGHT * getApplicationContext().getResources().getDisplayMetrics().density);
 
+		FrameLayout.LayoutParams params = (FrameLayout.LayoutParams) aspectRatioFrameLayout.getLayoutParams();
+		float density=getApplicationContext().getResources().getDisplayMetrics().density;
+		if(density>=2.0 && density<2.5) {
```

Comment:
@JideshNair I see the following conditions
Density = 1
Density > 1 && Density < 1.5
Density >= 2.0 && Density < 2.5
Density > 2.5

What about Density > 1.5 and Density < 2.0

Reply:
fixed